ext/hiera.yaml

```diff
@@ -2,14 +2,12 @@
 :backends:
   - yaml
 :hierarchy:
-  - defaults
-  - "%{clientcert}"
-  - "%{environment}"
-  - global
+  - "nodes/%{clientcert}"
```

Comment:
We should incorporate the suggested improvement from Nick Fagerlund (see ticket) and use `trusted.clientcert` instead of just `clientcert`.